### ISSUE TITLE
weird Gemini parachute fail (heatshield+pod strange drag)

### ISSUE THREAD

**Issue by Araym-KSP** [CobaltWolf/Bluedog-Design-Bureau#146]
Check Gemini's drag of reentry.

Mission started like a MOL... various orbital operation... then recovering of a stranded kerbal (mission grabbed after Gemini-MOL assembly was already in space, to recover her in low orbit, standard at ~80-85km) left me with monoprop fumes.
Managed a reentry with ~26-27 m/s Δv, with burn almost half orbit away from landing, very shallow trajectory (a profile I considered "very safe", to slow down most of the speed with atmosphere) .
Ejected Gemini SM and nose docking at ~50-45km (I was worried by an hi speed low orbit, carring a rescued kerbal, so I took advantage of the SM, already proven -another past mission- to be sturdy enough to aerobrake from Mun orbit to peri at 45km :P, to save ablator), reentry nominal untill low atmosphere...
... then weird things appened:

Capsule (already separated by all things not needed to land) seemed to not slow down a lot untill 5000m, then arrived "almost" at safe velocity for chute deploy (~270 m/s) at 2000-2500m, but suddenly building speed again: tried to pop up the chute, broken at around 280-290 m/s at ~1500m...
... (rage)quit (ALT-F4) of the game (I try to not F5 in career), to save the kerbals and reload them in orbit, to sent another mission with engineer + a couple of stock drogue chute more to bolt to the capsule with KIS + more fuel (it was a career mission) for a more standard "mechjeb" reentry a quarter of orbit away from landing.

Long story short?
Gemini pod drag seems weird (not the first time I felt it problematic) and very risky from very low orbit (under 85-80 km), and strangely, with shallow reentry that should bleed speed better, using more time in atmosphere (a profile I used a lot only for shuttle-alike vehicle without heatshields, rather my standard "plunge in" like always with pods+heatshields).
Standard Gemini (2x Kerbals) configuration, using parts from v 0.11 "Vanquero"

**Comment by jsolson** [comment]
Tagging issue #48

**Comment by jsolson** [comment]
Are you using FAR? I just did a 200x30 reentry with no issues. Ditched the SM in orbit, and hung on to the docking port until mach 1. I was at safe chute deployment speed around 6000 meters and did not accelerate. I even waited until 1000 meters to deploy the chute and *just* landed safely in the water. G's never went into the red on deployment.

I'll try a direct Mun return when I get a chance.

**Comment by Araym-KSP** [comment]
nope... stock atmosphere... 
The only things that could eventually comes in mind to me could be the "Stock Bug Fixes"  mod, as it has some cosmetic influence on parachutes deployment (spreading them, rather than clipping them all in the "up" direction when deployed)

What I noticed is that (as a paradox) higher orbit landings and steep reentries seem fine, but it starts to suffer from low orbit and shallow one (like the one that I tried from the low orbit where the game place the capsule for rescue missions, between 80-90 km)...
Maybe I hit just a very particulary case, as:
- the reentry burn was done on the opposite side of kerbin to the Kerbal Space Center at ~85km altitude, to use as the best the very low fuel I had (just those 26-27 m/s Δv left, if I remember correctly)
- i kept the SM in the atmo up to 50-45km (maybe influencing then the slow down, as the added weight could keep it faster)

**Comment by jsolson** [comment]
Tried deorbiting from 80km with 27dv (80x50 orbit). Hung on to the SM until 45 km. Safe to deploy chute by 7km. I can't get Gemini to misbehave on reentry.

**Comment by Cowboy-WY** [comment]
Just did 85kmx35km reentry. Using stock and BDB  only.

Entered atmosphere @ 2280m/s - Disconnected SM - oriented retrograde then turned off SAS
65km - 2295.2m/s
60km - 2311.8m/s
55km - 2322.8m/s
50km - 2329.9m/s
45km - 2326.4m/s
40km - 2303.2m/s
35km - 2248.1m/s
30km - 1953.1m/s
25km - 1754.6m/s
20km - 1442.1m/s
15km - 1019.5m/s
10km - 588.7m/s
9km - 508.1m/s
8km - 414m/s
7km - 322m/s - Chute displays safe

Deployed chute and touchdown was at about 6m/s

Did this three times with roughly the same numbers as results.  Seems to work for me under stock conditions.

**Comment by jsolson** [comment]
Ok I'm going to close this. Gemini seems to be safe.

**Comment by CobaltWolf** [comment]
or is it?!?
DUN
DUN
DUNNNNNNNNNNNNNNNNNNNNNNNNNNNNNn
